kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/ReactiveAdminClient.java

```diff
@@ -90,10 +90,15 @@ public static Mono<ReactiveAdminClient> create(AdminClient adminClient) {
   }
 
   private static SupportedFeature getSupportedUpdateFeatureForVersion(String versionStr) {
-    float version = NumberUtil.parserClusterVersion(versionStr);
-    return version <= 2.3f
-        ? SupportedFeature.ALTER_CONFIGS
-        : SupportedFeature.INCREMENTAL_ALTER_CONFIGS;
+    try {
+      float version = NumberUtil.parserClusterVersion(versionStr);
+      return version <= 2.3f
+          ? SupportedFeature.ALTER_CONFIGS
+          : SupportedFeature.INCREMENTAL_ALTER_CONFIGS;
+    } catch (Exception e) {
```

Comment:
I would catch NumberFormatException here instead of Exception

Reply:
okay done